folders.py

```diff
@@ -128,15 +129,22 @@ def get_folder(folder_id, data):
 
 def get_all_persistent_cache_files(path):
     """Get all files in PersistentCache storage with "start-group" marker."""
+    regex = ".*start-group.*"
```

Comment:
Note: [`re.match`](https://docs.python.org/3/library/re.html#re.match) matches the beginning of the line, so the trailing `.*` can be omitted. Also, if you use [`re.search`](https://docs.python.org/3/library/re.html#re.search) you can skip the preceding `.*` as well.

Reply:
Good catch, thank you!
I wrote this code a while back but it wasn't working. So I had another look at it the other day, and the quick fix was to add the '.*' at each end. But I should have checked the docs for re.